data-store/db-client/migrations/20231007213152_person.up.sql

```diff
@@ -0,0 +1,53 @@
+create schema person;
+create table person.base (
+    person_id  int generated always as identity primary key,
+    first_name text not null,
+    last_name  text not null,
+    email      text not null
+);
+
+-- przedstawiciel_biura_nagrody
+create table person.awards_representative (
+    awards_representative_id int generated always as identity primary key,
+    person_id                int unique not null,
+
+    constraint awards_representative_person_fk
+        foreign key (person_id) references person.base(person_id)
+);
+
+-- czlonek_jury
+create table person.jury_member (
+    jury_member_id int generated always as identity primary key,
+    person_id      int not null,
+    submission_id  int not null,
```

Comment:
Do you maybe remember why there's such field? I know it's in the base ERD, but can't recall, why would jury member be connected directly to the submission. If you also think this is a mistake, please remove it.

Reply:
Yes, yesterday that had me scratching my head too. I assumed that a person can be jury member only for the assigned submission - but i don't think that makes a lot of sense

Reply:
I don't think either